package.json

```
@@ -8,6 +8,9 @@
     "cz": "cz"
   },
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/styles": "^4.11.3",
```

Pas besoin de cette dep quand t'installes le core
Les icônes on va en avoir besoin ou tout est fourni sur les maquettes ?

```suggestion
    "@material-ui/icons": "^4.11.2",
```